kafka.go

```diff
@@ -0,0 +1,70 @@
+package main
+
```

Comment:
this doe not look so nice to me here. 
we should encapsulate this. i also do not like giving the close chanel to the 'outside'. we should have a struct or an interface